libkbfs/md_journal.go

```diff
@@ -475,7 +476,8 @@ func (j mdJournal) putMD(rmd BareRootMetadata) (MdID, error) {
 		return MdID{}, err
 	}
 
-	_, _, _, err = j.getMDAndExtra(id, true)
+	p := j.mdDataPath(id)
```

Comment:
nit: Since you don't use `p` again you can save a line by inlining it into the `Stat` call below.